src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -55,6 +55,12 @@
     //  "Configuration": "192.168.99.100:6379,allowAdmin=true", // Redis Configuration string.
     //  "InstancePrefix": "" // Optional prefix allowing a Redis instance to be shared by different applications.
     //},
+    //"OrchardCore_Security": {
+    //  "ContentSecurityPolicy": [],
+    //  "FrameOptions": "SAMEORIGIN",
+    //  "PermissionsPolicy": [ "fullscreen=self" ],
+    //  "ReferrerPolicy": "no-referrer"
+    //},
```

Comment:
These settings aren't applied.

Reply:
Do you mean when you uncomment them aren't applied?

Reply:
Yes.

Reply:
There was a bug because the settings reterieval was different for both AdminSettings and appsettings.json, it got fix

Reply:
This is still the case, the settings from appsettings are not applied.

Reply:
> I can record a GIF screenshot, I'm not why it doesn't applied at your end. Are you sure you call ConfigureSecuritySettings() in the web app?

My mistake, I didn't add a `ConfigureSecuritySettings()`, sorry about that.